feature/tabsflow/favourites/src/main/java/com/epmedu/animeal/favourites/FavouritesScreenUI.kt

```diff
@@ -79,4 +106,16 @@ private fun MoreScreenPreview() {
             )
         ) {}
     }
+}
+
+@AnimealPreview
+@Composable
+private fun MoreScreenEmptyPreview() {
```

Comment:
Can you rename it to FavouritesScreenEmptyPreview and the preview above to FavouritesScreenPreview? We missed it somehow